### DISCUSSION TITLE
Upgrade from weed version 3.54 to latest version 3.67

### DISCUSSION THREAD

**Discussion by CKHH** [seaweedfs/seaweedfs#5573] [Q&A] [answered]
Hi,

I've been running with weed v3.54 for at least 6 months now. I like to upgrade to latest v3.67, does it require specific upgrade procedures?

Thanks.

**Comment by CKHH** [comment]
Hi

Are there any specific procedures to upgrade SeaweedFS version?

I'm revisiting this topic, planning to upgrade version 3.54 to the latest version 3.79.

Thanks and regards.

**Comment by kmlebedev** [comment, marked as answer]
In general, there shouldn't be any issues with backward compatibility.
Unfortunately, for the sake of development speed, we do not have major and patch versions, as this greatly slows down the release of new features, so bugs can leak into releases. You should have your own testing process on test stands or unit tests of any critical functionality that you use and if you see an error, report it immediately.

Please treat this with understanding, as the main maintainer @chrislusf is the only one developing the project.
